Tests/Test_BatteryBalancing_Task.c

```diff
@@ -0,0 +1,63 @@
+/* Copyright (c) 2021 UT Longhorn Racing Solar */
+
+#include "BatteryBalancing.h"
+#include "Tasks.h"
+
+/******************************************************************************
+ * BatteryBalancing Task Test Plan
+ * 
+ * 1. Build the BPS code for the stm32f413 for this test file (see README.md)
+ * 2. Flash the BPS (see README.md)
+ * 3. Connect the BPS UART USB to your laptop and open PuTTY (or comparable)
+ * 4. Connect the BPS test board
+ * 5. Keep all modules at half voltage except one (of your choice) and keep that at full
+ * 6. Reset the BPS by pressing and releasing the reset button
+ * 7. Probe the balancing transistor to ensure it is turned on
+ * 8. Repeat 5-7 for different modules
+ ****************************************************************************/
+
```

Comment:
We will also need to actually read voltages from the minion boards by calling `Voltage_UpdateMeasurements()`, since the battery balancing task uses `Voltage_GetModuleMillivoltage()` to access voltage readings. `Voltage_GetModuleMillivoltage()` does not read from the minion boards. It reads from an array in the STM32F413's memory that holds the most recently read voltage data.

Reply:
In the event that this test doesn't work, I think we should consider refactoring the Battery Balancing library so that the Balancing_Balance() code could go into the thread and the helper functions could become non-static. This way testing each helper function from the Balancing library would become more convenient.